DocSum/tests/test_compose_on_rocm.sh

```diff
@@ -0,0 +1,190 @@
+    validate_services \
+        "${ip_address}:9000/v1/chat/docsum" \
+        "data: " \
+        "llm" \
+        "docsum-llm-server" \
+        '{"query":"Text Embeddings Inference (TEI) is a toolkit for deploying and serving open source text embeddings and sequence classification models. TEI enables high-performance extraction for the most popular models, including FlagEmbedding, Ember, GTE and E5."}'
+}
+
+function validate_megaservice() {
+    local SERVICE_NAME="mega-docsum"
+    local DOCKER_NAME="docsum-backend-server"
+    local EXPECTED_RESULT="embedding"
+    local INPUT_DATA="messages=Text Embeddings Inference (TEI) is a toolkit for deploying and serving open source text embeddings and sequence classification models. TEI enables high-performance extraction for the most popular models, including FlagEmbedding, Ember, GTE and E5."
+    local URL="${ip_address}:8888/v1/docsum"
+
+    local HTTP_STATUS=$(curl -s -o /dev/null -w "%{http_code}" -X POST -F "$INPUT_DATA" -H 'Content-Type: multipart/form-data' "$URL")
+    if [ "$HTTP_STATUS" -eq 200 ]; then
+        echo "[ $SERVICE_NAME ] HTTP status is 200. Checking content..."
+
+        local CONTENT=$(curl -s -X POST -F "$INPUT_DATA" -H 'Content-Type: multipart/form-data' "$URL" | tee "${LOG_PATH}"/"${SERVICE_NAME}".log)
+
+        if echo "$CONTENT" | grep -q "$EXPECTED_RESULT"; then
+            echo "[ $SERVICE_NAME ] Content is as expected."
+        else
+            echo "[ $SERVICE_NAME ] Content does not match the expected result: $CONTENT"
+            docker logs "${DOCKER_NAME}" >> "${LOG_PATH}"/"${SERVICE_NAME}".log
+            exit 1
+        fi
+    else
+        echo "[ $SERVICE_NAME ] HTTP status is not 200. Received status was $HTTP_STATUS"
+        docker logs "${DOCKER_NAME}" >> "${LOG_PATH}"/"${SERVICE_NAME}".log
+        exit 1
+    fi
+    sleep 1s
+}
+
+#function validate_frontend() {
+#    cd "$WORKPATH"/ui/svelte
+#    local conda_env_name="OPEA_e2e"
+#    export PATH=${HOME}/miniforge3/bin/:$PATH
+#    if conda info --envs | grep -q "$conda_env_name"; then
+#        echo "$conda_env_name exist!"
+#    else
+#        conda create -n ${conda_env_name} python=3.12 -y
+#    fi
+#    source activate ${conda_env_name}
+#
+#    sed -i "s/localhost/$ip_address/g" playwright.config.ts
+#
+#    conda install -c conda-forge nodejs -y
+#    npm install && npm ci && npx playwright install --with-deps
+#    node -v && npm -v && pip list
+#
+#    exit_status=0
+#    npx playwright test || exit_status=$?
+#
+#    if [ $exit_status -ne 0 ]; then
+#        echo "[TEST INFO]: ---------frontend test failed---------"
+#        exit $exit_status
+#    else
+#        echo "[TEST INFO]: ---------frontend test passed---------"
+#    fi
+#}
+
+function stop_docker() {
+    cd "$WORKPATH"/docker_compose/amd/gpu/rocm
+    docker compose stop && docker compose rm -f
+}
+
+function main() {
+
+    stop_docker
+
+#    if [[ "$IMAGE_REPO" == "opea" ]]; then build_docker_images; fi
```

Comment:
We should open the image build, so the images will be build based on the PR. Sometimes a PR changed the dockerfile and scripts, CI need to verify it on all the supported hardware.

Reply:
revert docker build, done